node/test_force.js

```diff
@@ -270,26 +274,24 @@ function createCompileApp(tmpDir, appType, os) {
 //
 function updatePluginRepo(tmpDir, pluginRepoDir, branch, os) {
     log('Updating cordova plugin at ' + branch, COLOR.green);
-    shelljs.pushd(pluginRepoDir);
-    runProcessThrowError(path.join('tools', 'update.sh') + ' -b ' + branch + ' -o ' + os);
```

Comment:
runProcessThrowError can take care of the pushd, popd if passed a directory as second argument